superglue/lib/action_creators/requests.ts

```diff
@@ -208,7 +209,8 @@ export function visit(
         }
         const isGet = fetchArgs[1].method === 'GET'
 
-        meta.suggestedAction = 'push'
+        meta.suggestedAction = suggestedAction
+
```

Comment:
Lets give the developer more control. In other words, if a `suggestedAction` was passed to this function, we use that regardless of the logic between 214 to 224. 

Now i'm thinking about the naming of things. Maybe the parameter on line 155 could be called `action` so we can say: if the `action` parameter was explicitly passed (defaults to push), the `suggestedAction` will always equal to that value.